compiler/ilgen/OMRIlBuilder.cpp

```diff
@@ -1005,6 +1005,31 @@ OMR::IlBuilder::convertTo(TR::DataType typeTo, TR::IlValue *v, bool needUnsigned
    return convertedValue;
    }
 
+static TR::ILOpCodes
+getCoercionOpcode(TR::DataType fromType, TR::DataType toType)
+   {
+   if (fromType == TR::Int32 && toType == TR::Float) return TR::ibits2f;
```

Comment:
This code isn't indented properly.

Also, I think a better place for this kind of utility function would be in `compiler/il/OMRILOps.[ch]pp` and just called from here.

Reply:
Done!